### PR TITLE
Create user

### PR DESCRIPTION
**Pull request by scmountain**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,6 +14,10 @@
   <% end %>
   <p>Cart: <%= @cart.total_count %></p>
   <%= button_to "View Cart", cart_path(session[:cart]), method: :get %>
+  <div class="navbar">
+    <%= button_to "Login", login_path, method: :get unless current_user %>
```
Comment (blackknight75) [inline app/views/layouts/application.html.erb]: Like the conditional to control what the user sees.

File: spec/features/users/00_authenticated_user_spec.rb [hunk 2]
```diff
@@ -2,14 +2,31 @@
...
+             username: "billygoat",
+             password: "pass",
+             address: "123 Any St, Denver, CO 80202",
+             email: "billygoat@gmail.com"
+            )
+
     visit root_path
-    click_on "Login"
+
+    within('.navbar') do
```
Comment (blackknight75) [inline spec/features/users/00_authenticated_user_spec.rb]: I liked that you wrapped the nav bar in a div class so it clicks the proper login button on sessions new erb.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -4,8 +4,13 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
```
Comment (blackknight75) [inline app/controllers/application_controller.rb]: can we take this out?

File: db/schema.rb [hunk 4]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170222165739) do
+ActiveRecord::Schema.define(version: 20170224032612) do
 
   # These are extensions that must be enabled in order to support this database
```
Comment (blackknight75) [inline db/schema.rb]: Lets get rid of this too.

File: db/schema.rb [hunk 5]
```diff
@@ -11,7 +11,7 @@
 #
```
Comment (blackknight75) [inline db/schema.rb]: One more deletion.

**Comment by brandonrandall** [comment]
nothing to review

**Pull request merged**